src/client/src/app/main-nav/main-nav.component.html

```diff
@@ -0,0 +1,88 @@
+<mat-sidenav-container class="sidenav-container">
+  <mat-sidenav #drawer class="sidenav" fixedInViewport
+      [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
+      [mode]="(isHandset$ | async) ? 'over' : 'side'"
+      [opened]="(isHandset$ | async) === true">
+    <mat-toolbar color="secondary"><mat-icon>group_work</mat-icon></mat-toolbar>
+    <mat-nav-list>
+      <!-- <a mat-list-item href="#">Home</a> -->
+
+      <a mat-list-item routerLink="home" (click)="drawer.toggle()"> <mat-icon>home</mat-icon></a>
+      
+      <!-- <a mat-list-item routerLink="games">Games</a> -->
+      <mat-menu #appMenu="matMenu" class="dropdown">
+        <button mat-menu-item routerLink="games" (click)="drawer.toggle()">All Games</button>
+        <button mat-menu-item routerLink="create-game" (click)="drawer.toggle()">Add Game</button>
+        <button mat-menu-item routerLink="join-game" (click)="drawer.toggle()">Play Game</button>
+      </mat-menu>
+      <button mat-button [matMenuTriggerFor]="appMenu">Games<mat-icon>unfold_more</mat-icon></button>
+
+      <a mat-list-item routerLink="create-merch" (click)="drawer.toggle()">Merch</a>
```

Comment:
I believe Aman has added some pages for merch where she needs direct access for now.

Reply:
updated